packages/grafana-ui/src/components/uPlot/config/setupUPlotConfig.ts

```diff
@@ -16,14 +18,14 @@ type SetupConfigParams = {
   onUPlotClick: () => void;
   setFocusedSeriesIdx: Dispatch<SetStateAction<number | null>>;
   setFocusedPointIdx: Dispatch<SetStateAction<number | null>>;
-  setCoords: Dispatch<SetStateAction<CartesianCoords2D | null>>;
+  setCoords: Dispatch<SetStateAction<{ viewport: CartesianCoords2D; canvas: CartesianCoords2D } | null>>;
   setHover: Dispatch<SetStateAction<HoverEvent | undefined>>;
   isToolTipOpen: MutableRefObject<boolean>;
 };
 
 // This applies config hooks to setup tooltip listener. Ideally this could happen in the same `prepConfig` function
 // however the GraphNG structures do not allow access to the `setHover` callback
-export const setupConfig = ({
+export const setupUPlotConfig = ({
```

Comment:
Since this function is only about enabling the tooltip interaction I suggest renaming it to `addTooltipSupport` - and this should result in the file name change too.

Reply:
I renamed it as suggested